developers-api/src/main/java/com/quarksy/api/DevelopersResource.java

```diff
@@ -62,7 +78,11 @@ public Developer updateDevById(@PathParam("id") String id, Developer dev)
     public Response delete(@PathParam("id") String id)
     {
         logger.info("Delete Developer By ID");
-        return service.removeDevById(id);
+        try {
+            return service.removeDevById(id);
+        } catch (Exception e) {
+            throw new WebApplicationException(Response.status(BAD_REQUEST).entity("An error has occurred.").build());
```

Comment:
I think we should return error message "No developer with that ID was found' or something similar.